.github/workflows/gradle.yml

```diff
@@ -97,6 +97,8 @@ jobs:
         - "6.9"
         - "7.0.2"
         - "7.1"
+        - "7.3"
```

Comment:
Added verification on the two newer gradle major releases. I also tried 8.0 but it does not work, so left it out for now.